Generic_CCVA.Rmd

```diff
@@ -637,3 +686,37 @@ color_list = c("#882314", "#C0532B", "#CF932C", "#674D53", "#8C86A0", "#724438",
                "#0067A2", "#DFCB91", "#CB7223", "#289A84", "#7FA4C2", "#AF7E56")
 
 ```
+
+
+2. Example Tables
+```{r, eval = TRUE, echo = FALSE, message = FALSE, warning = FALSE}
+
+kable(annual_stats %>%
+        dplyr::select(c("cf","aet_in_mean",
```

Comment:
You can combine `rename()` and `select()` functions such that this could be : 

`dplyr::select(AET = aet_in_mean, Runoff = runoff_in_mean .....)`

You also shouldn't need to put it within `c()` or column name in quotes.